coordinator-service/src/disk-coordinator.ts

```diff
@@ -135,6 +139,22 @@ export class DiskCoordinator implements Coordinator {
         return true
     }
 
+    tryUnlockChunk(chunkId: string, participantId: string): boolean {
+        const ceremony = this._readDb()
+
+        const chunk = DiskCoordinator._getChunk(ceremony, chunkId)
+        if (chunk.lockHolder !== participantId) {
+            throw new Error(
+                `${participantId} does not hold lock on chunk ${chunkId}`,
+            )
+        }
+
+        chunk.lockHolder = null
+        chunk.metadata.lockHolderTime = null
```

Comment:
the original semantics of this field are the last time `lockHolder` is updated.  Analogous to mtime on a file.  The other "time" fields are the same (when the corresponding field is update, a new timestamp is recorded).

Reply:
That's a good point. I'll change it to keep the semantics.